src/fileformat/types/sec_seg/sec_seg.cpp

```diff
@@ -521,6 +536,27 @@ void SecSeg::setMemory(bool sMemory)
 	isInMemory = sMemory;
 }
 
+/**
+ * Compute entropy of section data in <0,1>
+ */
+void SecSeg::computeEntropy()
+{
+	if (!loaded)
+	{
+		return;
+	}
+	
+	auto data = reinterpret_cast<const uint8_t *>(bytes.data());
```

Comment:
Do we need a `reinterpret_cast` cast for this? Doesn't a `static_cast` suffice?

Reply:
I belive not https://stackoverflow.com/questions/658913/c-style-cast-from-unsigned-char-to-const-char